stdlib/array.ml

```diff
@@ -13,6 +13,9 @@
 (*                                                                        *)
 (**************************************************************************)
 
+(* An alias for the type of arrays. *)
+type nonrec 'a t = 'a array
```

Comment:
nonrec is useless here (and in other modules as well).

Reply:
Thanks.  I've removed the useless `nonrec` keywords.